client/app/components/ApplicationArea/ApplicationLayout/DesktopNavbar.jsx

```diff
@@ -71,39 +71,35 @@ export default function DesktopNavbar() {
   const canCreateAlert = currentUser.hasPermission("list_alerts");
 
   return (
-    <div className="desktop-navbar" role="navigation">
+    <nav className="desktop-navbar" role="navigation">
       <NavbarSection className="desktop-navbar-logo">
         <div role="menuitem">
-          <Link href="./" tabIndex={0}>
+          <Link href="./">
             <img src={logoUrl} alt="Redash" />
           </Link>
         </div>
       </NavbarSection>
 
       <NavbarSection>
         {currentUser.hasPermission("list_dashboards") && (
-          <Menu.Item
-            tabIndex={-1}
-            key="dashboards"
-            role="menuitem"
-            className={activeState.dashboards ? "navbar-active-item" : null}>
-            <Link href="dashboards" tabIndex={0}>
+          <Menu.Item key="dashboards" role="menuitem" className={activeState.dashboards ? "navbar-active-item" : null}>
```

Comment:
Isn't the role of `menuitem` [built into `Menu.Item`](https://github.com/react-component/menu/blob/master/src/MenuItem.tsx#L189)?

Reply:
It is! Sorry for this!